client/src/App.js

```diff
@@ -33,34 +33,46 @@ class App extends Component {
                 isYogaAdmin: false,
                 setAccess: this.setAccess,
 
-                lang: 'hu',
+                lang: localStorage.getItem('lang'),
                 dictionary: dictionaryList[localStorage.getItem('lang')],
                 changeLang: this.changeLang
             },
 
             dataContext: {
-                userName: 'unknown',
+                userName: {
+                    firstName: '',
+                    lastName: '',
+                    fullName: '',
+                    nameOrder: localStorage.getItem('lang') === 'hu' ? -1 : 1
```

Comment:
Magic numbers? Why not boolean?

Reply:
I considered 1 and -1 better for the representation of an order than a boolean value. If it would be readable to you as a boolean, I can change it.